TST: `scipy/fft/tests/test_basic.py::test_multiprocess[rfft]` too slow

This test has crossed the fail-slow threshold 9 times in the past month:

<img width="764" height="559" alt="Image" src="https://github.com/user-attachments/assets/a35c88e0-c1f3-48b7-b68f-11da65709e52" />

https://actions-search.scicftest.com/actions/scipy/scipy/

```
FAILED scipy/fft/tests/test_basic.py::test_multiprocess[rfft] - Test passed but took too long to run: Duration 1.1592951430002358s > 1.0s
```

https://github.com/scipy/scipy/actions/runs/28334351617/job/83937867538

--- a/scipy/fft/tests/test_basic.py
+++ b/scipy/fft/tests/test_basic.py
@@ -481,6 +481,7 @@ class TestFFTThreadSafe:
         self._test_mtsame(fft.ihfft, a, xp=xp)
 
 
+@pytest.mark.fail_slow(2)
 @pytest.mark.parametrize("func", [fft.fft, fft.ifft, fft.rfft, fft.irfft])
 def test_multiprocess(func):
     # Test that fft still works after fork (gh-10422)

